### PR TITLE
changed EIFSAttributes in saml_helper to GlobalDirectoryAttributes

### PR DESCRIPTION
**Pull request by DGabriel-CCPO**
Title says it all

### CODE REVIEW

**Review by jeffsegal-ccpo — CHANGES_REQUESTED** [review]

File: atat/routes/saml_helpers.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
 # The SAML attributes happen to have names in a URL format
 # This does not mean that there is communication via HTTP
-class EIFSAttributes:
+class GlobalDirectoryAttributes:
```
[the remark this answers is not in the document]
Comment (DGabriel-CCPO) [reply atat/routes/saml_helpers.py:35]: Has been changed in the code.

[the remark this answers is not in the document]
Comment (jeffsegal-ccpo) [reply atat/routes/saml_helpers.py:35]: Don't see it yet - did you forget to do a push?

[the remark this answers is not in the document]
Comment (DGabriel-CCPO) [reply atat/routes/saml_helpers.py:35]: Screen Shot 2021-04-23 at 2 48 51 PM
I don't know why it didn't update the pull request

[the remark this answers is not in the document]
Comment (jeffsegal-ccpo) [reply atat/routes/saml_helpers.py:35]: M ay be because it's behind `develop` itself? 

image

**Comment by jasonburkert-ccpo** [comment]
@jeffsegal-ccpo Do you approve of the requested changes?

**Review by jeffsegal-ccpo — APPROVED** [review]

**Pull request merged**